src/main/java/driver/LocalDriverStategy.java

```diff
@@ -0,0 +1,79 @@
+package driver;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.jetbrains.annotations.Contract;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import utility.PropertyProvider;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class LocalDriverStategy {
+
+    private static final String WEBDRIVER_GECKO_DRIVER = "webdriver.gecko.driver";
+    private static final String GECKODRIVER_EXE_PATH = ".\\geckodriver\\geckodriver.exe";
+    private static final String WEBDRIVER_CHROME_DRIVER = "webdriver.chrome.driver";
+    private static final String CHROMEDRIVER_EXE_PATH = ".\\chromedriver\\chromedriver.exe";
+    private static final Logger log = LogManager.getRootLogger();
+    private static WebDriver driver;
+    //private String browser = "chrome";
+
+    public LocalDriverStategy(){
```

Comment:
Mmm... There is no reasons to have public non-args ctor, because it exists without declaration by default. I think you wanted private ctor to restrict creation of this class